geode-lucene/src/main/java/org/apache/geode/cache/lucene/LuceneIndex.java

```diff
@@ -19,39 +19,62 @@
 
 import org.apache.lucene.analysis.Analyzer;
 
-import org.apache.geode.annotations.Experimental;
-
 
 /**
  * <p>
- * An Lucene index is built over the data stored in a GemFire Region.
+ * LuceneIndex represents the Lucene index created over the data stored in Apache Geode regions. The
+ * Lucene indexes are maintained automatically by Apache Geode whenever the entries are updated in
+ * the associated regions. Lucene Indexes are created using {@link LuceneService#createIndexFactory}
+ * by specifying the Lucene index name, the region associated with the Lucene index, the fields on
```

Comment:
maybe just add the word "and" before the last phrase here ... "and the fields on ..."